Formula/djl-bench.rb

```diff
@@ -0,0 +1,32 @@
+class DjlBench < Formula
+  desc "Command-line tool that helps to benchmark the model on different platforms"
+  homepage "https://github.com/deepjavalibrary/djl/tree/master/extensions/benchmark"
+  url "https://publish.djl.ai/djl-bench/0.12.0/benchmark-0.12.0.tar"
```

Comment:
Is there no compressed tarball?

Reply:
I don't think we have a further compressed version.